src/frontend/components/UI/HowLongToBeat/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { HowLongToBeatEntry } from 'howlongtobeat'
+import React, { useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import './index.scss'
+
+// react component with 3 square with how long to beat info
+
+type Props = {
+  title: string
+}
+
+export default function HowLongToBeat({ title }: Props) {
+  const [howLongToBeatInfo, setHowLongToBeatInfo] =
+    React.useState<HowLongToBeatEntry | null>(null)
+  const { t } = useTranslation('gamepage')
+
+  useEffect(() => {
+    window.api.getHowLongToBeat(title).then((howLongToBeatInfo) => {
+      if (howLongToBeatInfo) {
+        setHowLongToBeatInfo(howLongToBeatInfo)
+      }
+    })
+  }, [title])
+
+  if (!howLongToBeatInfo) {
+    return null
+  }
+
+  const { gameplayMain, gameplayMainExtra, gameplayCompletionist } =
+    howLongToBeatInfo || {}
+
+  return (
+    <>
+      <h5>{t('howLongToBeat', 'How Long To Beat')}</h5>
+      <div className="howLongToBeat">
+        <div className="howLongToBeat__square">
+          <div className="howLongToBeat__square__title">
+            {t('how-long-to-beat.main-story', 'Main Story')}
+          </div>
+          <div className="howLongToBeat__square__value">
+            {gameplayMain || '?'} {t('hours', 'Hours')}
+          </div>
+        </div>
+        <div className="howLongToBeat__square">
+          <div className="howLongToBeat__square__title">
+            {t('how-long-to-beat.main-plus', 'Main')} +{' '}
+            {t('how-long-to-beat.main-extras', 'Extras')}
```

Comment:
I think it's better to have this string to be translated as a whole `Main + Extras`, I don't know about grammar rules for other languages but maybe these are the other way around in some language or something different is more common than the `+` symbol (like &)

We don't need the 2 string separated anyway

Reply:
Done